public/app/features/dashboard/dashgrid/PanelHeader/PanelHeader.tsx

```diff
@@ -44,7 +44,7 @@ export const PanelHeader: FC<Props> = ({ panel, error, isViewing, isEditing, dat
         error={error}
       />
       <div className={className}>
-        <PanelHeaderMenuTrigger aria-label={selectors.components.Panels.Panel.title(title)}>
+        <PanelHeaderMenuTrigger data-testid={selectors.components.Panels.Panel.title(title)}>
```

Comment:
The errors this is throwing could have to do with the title not being unique

Reply:
Trying this with a screen reader makes me think it isn't needed regardless. The panel is announced and the heading is announced.

Reply:
if this isn't needed, then can we remove it? agreed it will cause errors if the panel title isn't unique